src/VisualStudio/IntegrationTest/IntegrationTests/AbstractIntegrationTest.cs

```diff
@@ -63,6 +63,12 @@ public virtual async Task InitializeAsync()
         /// </summary>
         public virtual Task DisposeAsync()
         {
+            if (VisualStudio?.Editor.IsCompletionActive() ?? false)
+            {
+                // Make sure completion isn't visible
+                VisualStudio.SendKeys.Send(VirtualKey.Escape);
```

Comment:
📝 This is avoiding a crash in the editor code for the current release.
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/801435